packages/opentelemetry-node-sdk/src/instrumentation/PluginLoader.ts

```diff
@@ -79,10 +110,27 @@ export class PluginLoader {
       }
 
       // Enable the require hook.
-      hook(modulesToHook, (exports, name, baseDir) => {
+      hook(modulesToHook, { internals: true }, (exports, name, baseDir) => {
```

Comment:
This is the new entry point into the internal stuff. It causes this to function to trigger for every internal file loaded in the module. This is probably not good for perf even though the module is immediately returned as is when there is no patching to be done.